packages/shared/lib/services/connection.service.ts

```diff
@@ -528,9 +528,9 @@ class ConnectionService {
     ): Promise<{ id: number; connection_id: string; provider: string; created: string; metadata: Metadata; active_logs: ActiveLogIds }[]> {
         const queryBuilder = db.knex
             .from<Connection>(`_nango_connections`)
-            .leftJoin(ACTIVE_LOG_TABLE, function () {
-                this.on('_nango_connections.id', '=', `${ACTIVE_LOG_TABLE}.connection_id`).andOnVal(`${ACTIVE_LOG_TABLE}.active`, true);
-            })
+            .joinRaw(
+                `LEFT JOIN LATERAL (SELECT * FROM ${ACTIVE_LOG_TABLE} WHERE _nango_connections.id = ${ACTIVE_LOG_TABLE}.connection_id AND ${ACTIVE_LOG_TABLE}.active = true LIMIT 1) ${ACTIVE_LOG_TABLE} ON true`
```

Comment:
not sure the active logs belong into this query but can you add a comment to explain why we are doing a lateral join subquery.

Reply:
Added comment with https://github.com/NangoHQ/nango/pull/2263/commits/16291a900206a59a5f0e312338eb711a0ae1d0a6